src/content/docs/data-apis/manage-data/drop-data-using-nerdgraph.mdx

```diff
@@ -36,7 +36,7 @@ Learn more about dropping data in this video (7:09 minutes):
 
 Besides creating drop-data rules, other ways to minimize unwanted data include:
 
-* If you want to drop APM metric timeslice data, you can use [metric normalization rules](/docs/new-relic-one/use-new-relic-one/ui-data/metric-normalization-rules).
+* If you want to drop [APM metric timeslice data](docs/data-apis/understand-data/metric-data/query-apm-metric-timeslice-data-nrql), you can use [metric normalization rules](/docs/new-relic-one/use-new-relic-one/ui-data/metric-normalization-rules), you can't drop metric timeslice data with drop rules. For more information about the difference between metric timeslice data and dimensional metrics see [this doc](/docs/data-apis/understand-data/new-relic-data-types/#timeslice-data).
```

Comment:
Suggestion: 

If you want to drop [APM metric timeslice data](/docs/data-apis/understand-data/metric-data/query-apm-metric-timeslice-data-nrql), you can use [metric normalization rules](/docs/new-relic-one/use-new-relic-one/ui-data/metric-normalization-rules). However, you can't drop metric timeslice data with drop rules. For more information on difference between metric timeslice data and dimensional metrics, refer [New Relic data types.](/docs/data-apis/understand-data/new-relic-data-types/#timeslice-data)

Reply:
looks good to me, thank you